depends/packages/libmultiprocess.mk

```diff
@@ -4,6 +4,16 @@ $(package)_download_path=$(native_$(package)_download_path)
 $(package)_file_name=$(native_$(package)_file_name)
 $(package)_sha256_hash=$(native_$(package)_sha256_hash)
 $(package)_dependencies=native_$(package) capnp
```

Comment:
In commit "build: Fix `libmultiprocess` cross-compiling to Linux hosts" (c0f5cc14ef9fae2b2de4222ee061729629ebb6b4)

In a different PR, could probably drop `native_$(package)` as a dependency here. The cross compiled libmultiprocess packages doesn't actually depend on native libmultiprocess package for anything. (This is different from the cross compiled capnp which does depend on the native capnp for codegen tools used during cross-compilation).